js/sectionView.js

```diff
@@ -0,0 +1,21 @@
+var viewSection = {};
+
+viewSection.handleMainNav = function(){
+  $('nav').on('click', '.tab', function(){
+    var target = $(this);
+    $('.main-section').hide();
+    $('.tab').removeClass('active');
+    $('.main-section').each(function(index){
```

Comment:
There's a better way to handle this than a loop. If you have the data content, and it matches the id...how would you select an id? (hint: the same way you'd select it in CSS!)